pkg/controller/rbac_jobs.go

```diff
@@ -86,9 +87,18 @@ func (c *StashController) ensureCronJobRoleBinding(resource *core.ObjectReferenc
 		Namespace: resource.Namespace,
 	}
 
+	bcObj, err := c.stashClient.StashV1beta1().BackupConfigurations(resource.Namespace).Get(resource.Name, metav1.GetOptions{})
```

Comment:
We need to set workload as owner for `BackupSidecarRoleBinding`, `RestoreInitContainerRoleBinding`, `BackupJobRoleBinding`.

Not `CronJobRoleBinding`. CronJob has no relation with workload.

Reply:
fixed